tests/test_conf_minimal.py

```diff
@@ -95,61 +94,91 @@ def test_ell():
 
-# @pytest.mark.slow
+
 def test_xi_block():
+    # Test made independent of pickled objects
     tracer_comb1 = ('lens0', 'lens0')
     tracer_comb2 = ('lens0', 'lens0')
 
     print(f"Checking covariance block. \
           Tracer combination {tracer_comb1} {tracer_comb2}")
+    s = tjp0.cl_data
+
+    ell = tjp0.ell
+    ccltr = ccl_tracers['lens0']
+    cl = ccl.angular_cl(cosmo, ccltr, ccltr, ell) + tracer_Noise['lens0']
+
+    fsky = tjp0.fsky
+    dl = np.gradient(ell)
+    cov = np.diag(2 * cl**2 / (4 * np.pi * fsky))
+
+    tjp0.WT = tjp0.wt_setup(tjp0.ell, tjp0.theta)
+    s1_s2 = tjp0.get_cov_WT_spin(tracer_comb=tracer_comb1)
+    th, cov = tjp0.WT.projected_covariance2(l_cl=ell, s1_s2=s1_s2,
+                                            s1_s2_cross=s1_s2, cl_cov=cov)
 
     gcov_xi_1 = tjp0.cl_gaussian_cov(tracer_comb1=tracer_comb1,
                                      tracer_comb2=tracer_comb2,
                                      ccl_tracers=ccl_tracers,
                                      tracer_Noise=tracer_Noise,
                                      two_point_data=tjp0.cl_data,
                                      do_xi=True)
-    np.testing.assert_allclose(gcov_xi_1['final'], ref_covnobin)
-    return
+    np.testing.assert_allclose(gcov_xi_1['final'], cov)
 
 
-# @pytest.mark.slow
 def test_cl_block():
-
+    # Test made independent of pickled objects
     tracer_comb1 = ('lens0', 'lens0')
     tracer_comb2 = ('lens0', 'lens0')
 
     print(f"Checking covariance block. \
           Tracer combination {tracer_comb1} {tracer_comb2}")
 
+    s = tjp0.cl_data
+
+    ell = tjp0.ell
+    ccltr = ccl_tracers['lens0']
+    cl = ccl.angular_cl(cosmo, ccltr, ccltr, ell) + tracer_Noise['lens0']
+
+    fsky = tjp0.fsky
+    dl = np.gradient(ell)
+    cov = np.diag(2 * cl**2 / ((2 * ell + 1) * fsky * dl))
+    lb, cov = bin_cov(r=ell, r_bins=tjp0.ell_edges, cov=cov)
+
     gcov_cl_1 = tjp0.cl_gaussian_cov(tracer_comb1=tracer_comb1,
                                      tracer_comb2=tracer_comb2,
                                      ccl_tracers=ccl_tracers,
                                      tracer_Noise=tracer_Noise,
                                      two_point_data=tjp0.cl_data,
                                      do_xi=False)
     np.testing.assert_allclose(gcov_cl_1['final_b'],
-                               ref_cov0cl[:24, :24])
-
+                               cov)
 
-@pytest.mark.slow
-def test_cl_cov():
-    print("Comparing Cl covariance (840 data points)")
-    gcov_cl = tjp0.get_all_cov()
-    np.testing.assert_allclose(gcov_cl,
-                               ref_cov0cl[:, :])
-
-
-@pytest.mark.slow
-def test_xi_cov():
-    print("Comparing xi covariance (700 data points)")
-    covall_xi = tjpcov.get_all_cov(do_xi=True)
-    np.testing.assert_allclose(gcov_cl,
-                               ref_cov0xi[:, :])
 
+# Note: These tests will be innecessary after the refactoring because the
```

Comment:
I'd rather leave them there until we reorganize the code because the fsky full covariance is not built with the same function as the NaMaster and the SSC ones. That way I remind myself (and others) that it is different and untested at the moment (although it worked before so it should be fine).